incubator/azuremonitor-containers/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 2.0.0-3
```

Comment:
should this be updated as well?

Reply:
No. We have versions for all our components.